src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/util/DateTimeHelper.kt

```diff
@@ -31,10 +36,10 @@ class DateTimeHelper {
       return date?.format(formatter) ?: ""
     }
 
-    fun splitDateTime(dateTime: String?): Pair<String, String> {
-      val parts = dateTime?.split("T")
-
-      return if (parts != null) Pair(parts[0], parts[1].substring(0, 8)) else Pair("", "")
+    fun splitDateTime(dateTime: LocalDateTime?): Pair<String?, String?> {
+      val formatterDate = DateTimeFormatter.ofPattern("dd/MM/yyyy")
+      val formatterTime = DateTimeFormatter.ofPattern("HH:mm:ss")
```

Comment:
Is there a requirement to show seconds? Seems very specific

Reply:
I can add milliseconds as well if we want to be even more specific?

Reply:
Seconds removed